lib/src/builders/statement/break.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:analyzer/analyzer.dart';
+import 'package:analyzer/dart/ast/standard_ast_factory.dart';
+import 'package:code_builder/src/builders/shared.dart';
+import 'package:code_builder/src/builders/statement.dart';
+import 'package:code_builder/src/tokens.dart';
+
+/// Represents a break statement AST.
+abstract class BreakStatementBuilder implements StatementBuilder {
```

Comment:
Can we just replace this with a singleton instance, i.e.

const breakStatement = const _BreakStatementBuilder();